README_PYPI.md

```diff
@@ -0,0 +1,99 @@
+<p>
+   <img src="https://raw.githubusercontent.com/Mstrutov/Desbordante/main/images/desbordante-preview.png"/>
+</p>
+
+---
+
+# desbordante: high-performance data profiler
+
+## What is it?
+
+**desbordante** is a high-performance data profiler oriented towards exploratory data analysis
+
+Website: https://desbordante.unidata-platform.ru/
+
+## Table of Content
+
+- [Main Features](#main-features)
+- [Usage Example](#usage-example)
+- [Installation](#installation)
+- [Installation from sources](#installation-from-sources)
+- [Cite](#cite)
+- [Contacts and Q&A](#contacts-and-qa)
+
+# Main Features
+
+**desbordante** can discover and validate a range of data patterns, such as
+
+1. Functional dependencies, both exact and approximate (discovery and validation)
+2. Conditional functional dependencies (discovery)
+3. Metric functional dependencies (validation)
+4. Fuzzy algebraic constraints (discovery)
+5. Association rules (discovery)
+
+Package is just a bindings for the core c++ library, so it is currently **2-10 times faster** than the
+competitors, depending on the algorithm and table.
```

Comment:
```suggestion
This package uses the library of the platform named Desbordante, which is written in C++.
Depending on the algorithm and dataset, it may be **2-10 times faster** than the alternatives.
```